### PR TITLE
Azure: Remove quotes from tags in test VHD script

### PR DESCRIPTION
**Pull request by willie-yao**
What this PR does / why we need it:
This PR removes quotes when referencing `$TAGS` which caused tags to be incorrectly formatted for the az cli.

Which issue(s) this PR fixes (optional, in fixes #<issue number>(, fixes #<issue_number>, ...) format, will close the issue(s) when PR gets merged): Fixes #

**Additional context**
Add any other context for the reviewers

### CODE REVIEW

File: images/capi/packer/azure/.pipelines/test-vhd.yaml [hunk 1]
```diff
@@ -53,7 +53,7 @@ jobs:
       AZURE_LOCATION=$(az storage account show --name "${STORAGE_ACCOUNT_NAME}" --query '[location]' -o tsv)
 
       # Create the resource group
-      az group create --name "${RESOURCE_GROUP}" --location "${AZURE_LOCATION}" --tags "${TAGS}"
+      az group create --name "${RESOURCE_GROUP}" --location "${AZURE_LOCATION}" --tags ${TAGS}
```
Comment (CecileRobertMichon) [inline images/capi/packer/azure/.pipelines/test-vhd.yaml:56]: can you please add a comment above to explain why tags isn't surrounded by quotes so we know not to revert this change if we start running shell check later

Comment (willie-yao) [reply images/capi/packer/azure/.pipelines/test-vhd.yaml:57]: Added!

**Review by CecileRobertMichon — COMMENTED** [review]
/lgtm
/assign @mboersma

**Review by mboersma — APPROVED** [review]
/lgtm
/approve

**Pull request merged**